### ISSUE TITLE
Persistency of timeseries data deployed on Heroku

### ISSUE THREAD

**Issue by DBertazioli** [heartexlabs/label-studio#5675]
Hello, thanks for your amazing work!

I've deployed Labelstudio on Heroku following the link from your main page, and it comes automatically with a postgresql database (awesome!)

Now, when I load through the local import interface a timeseries csv to be annotated, after the app goes down cause not used, the annotations will persist (are stored in the database) but apparently not the input data (see the "undefined" error below). Therefore, there is no way to continue annotating a task that was started earlier, or reviewing it.

![image](https://github.com/HumanSignal/label-studio/assets/46977141/33d2361e-c01b-412c-acdd-e4dd2dfc23dd)

Is there any way to fix this, or is this behaviour intended?

Thanks in advance,
best
Dario

**Comment by farioas** [comment]
Hi,
Please check this official article: https://help.heroku.com/K1PPS2WM/why-are-my-file-uploads-missing-deleted-from-the-application
TLDR; it's impossible to configure persistence storage in Heroku, so far we already support a recommended way of persistence (by Heroku) as S3 buckets. Check our article: https://labelstud.io/guide/storage.html

Alternative (hardcore) way is to configure S3 persistent storage, in that case you can use heroku dyno as-is - it to follow this guide: https://labelstud.io/guide/persistent_storage.html#Configure-the-S3-bucket. Once you'll create S3 bucket, just pass these env variables to dyno instance and restart it:
```
STORAGE_TYPE=s3
STORAGE_AWS_ACCESS_KEY_ID="<YOUR_ACCESS_KEY_ID>"
STORAGE_AWS_SECRET_ACCESS_KEY="<YOUR_SECRET_ACCESS_KEY>"
STORAGE_AWS_BUCKET_NAME="<YOUR_BUCKET_NAME>"
STORAGE_AWS_REGION_NAME="<YOUR_BUCKET_REGION>"
STORAGE_AWS_FOLDER=""
```
